srtp/srtp.c

```diff
@@ -1482,6 +1786,16 @@ srtp_unprotect_aead (srtp_ctx_t *ctx, srtp_stream_ctx_t *stream, int delta,
     }
   }
 
+  if (xtn_hdr && stream->rtp_xtn_hdr_cipher) {
+    /*
+     * extensions header encryption RFC 6904
+     */
+    status = srtp_process_header_encryption(stream, xtn_hdr);
```

Comment:
Strange, it shows up for me on https://github.com/cisco/libsrtp/pull/128/commits